ee/vellum_cli/push.py

```diff
@@ -50,6 +51,9 @@ def push_command(
     workflow = BaseWorkflow.load_from_module(workflow_config.module)
     workflow_display = get_workflow_display(base_display_class=VellumWorkflowDisplay, workflow_class=workflow)
     exec_config = workflow_display.serialize()
+    exec_config["runner_config"] = {
+        "sdk_version": metadata.version("vellum-ai"),
```

Comment:
[blocking] could we add the image params from the default runner config for now? as it stands the runner config will save without it: https://github.com/vellum-ai/vellum/blob/main/django/app/api/workflows/utils.py#L72-L75

Reply:
I put the defaults in the RunnerConfigSerializer